mattermost-plugin/server/boards/configuration_test.go

```diff
@@ -55,7 +55,7 @@ func TestOnConfigurationChange(t *testing.T) {
 	basePlugins[PluginName][SharedBoardsName] = true
 
 	baseFeatureFlags := &serverModel.FeatureFlags{
-		BoardsFeatureFlags: "Feature1-Feature2",
+		// BoardsFeatureFlags: "Feature1-Feature2", // TODO: add feature flag for boards
```

Comment:
We ne longer need a feature flag given that it will not be embedded in the product. The feature flag was a way to disable the product within the MPA. Given that now it is a uploaded plugin, it can be disabled by disabling the plugin. 

We can remove this feature flag.

Reply:
Okay, removed it.